### PR TITLE
Viscomm Teaser Done

### PR DESCRIPTION
**Pull request by Ahmed-I-Abdullah**

### CODE REVIEW

File: Recruit-Training/Advanced-Recruit-Training/Viscomm-Teaser/Viscomm-Teaser-Training/src/InformationParser.cpp
Comment (Hesham-E) [inline Recruit-Training/Advanced-Recruit-Training/Viscomm-Teaser/Viscomm-Teaser-Training/src/InformationParser.cpp:111]: Hey! Great job on this! I am very impressed overall. There's just one minor thing that I wanted to mention here. Try to write your if-statements using the same format (ie. using curly braces for all of them) since it helps with readability.

Comment (Ahmed-I-Abdullah) [reply Recruit-Training/Advanced-Recruit-Training/Viscomm-Teaser/Viscomm-Teaser-Training/src/InformationParser.cpp:111]: Thank you! I will keep that in mind and update it in the next commit.

**Review by DatLam25 — APPROVED** [review]
Wait for another approval before closing this PR. Remember to not merge. Awesome work!

**Review by Hesham-E — APPROVED** [review]
Looks good to me as well!

**Pull request closed without merging**
